### ISSUE TITLE
TypeScript declaration

### ISSUE THREAD

**Issue by gfmio** [square/find-yarn-workspace-root#12]
Hi!

Thanks for this useful package. I'm using it with TypeScript, but there are no declarations for it yet. Can I add them?

The only required steps are to add the declaration file `index.d.ts` and add the `types` field to the `package.json` (`{ "types": "index.d.ts" }`. The declaration file should contain:

```ts
declare function findWorkspaceRoot(initial?: string | null | undefined): string | null;
export = findWorkspaceRoot;
```

Can I submit a pull request? 

Thanks!

**Comment by eventualbuddha** [comment]
I think that’d be great. Thanks!

**Comment by gfmio** [comment]
Just added a PR with the declaration file (#13).

**Comment by gfmio** [comment]
Closed by #13 and in package version 1.2.1.
